spec/factories/installment.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
+  factory :installment do
+    digit = Faker::Number.digit
+    value = Faker::Number.decimal(digit.to_i, 2)
+
+    value { value }
```

Comment:
I think that it's better to use Faker::Number.positive.round(2) to set the value